src/app/components/app.component.ts

```diff
@@ -36,10 +36,10 @@ import {Home} from './home/home';
   `
 })
 @RouteConfig([
-  { path: '/', component: Home, name: 'Index' },
-  { path: '/home', component: Home, name: 'Home' }
+  { path: '/', component: HomeCmp, name: 'Index' },
+  { path: '/home', component: HomeCmp, name: 'Home' }
 ])
-export class App {
+export class AppCmp {
```

Comment:
^ I like that. At end of day, I like to use what is universally found in the community. It just makes bringing new people into a project easier because they see what they are used to and have seen everywhere else.

So I like that personally, but don't want to use it if my project would be the only one using it ;)